cmd/git-sync/webhook.go

```diff
@@ -22,8 +22,15 @@ type Webhook struct {
 	Backoff time.Duration
 }
 
-func (w *Webhook) Do() error {
+type webhookRepoInfo struct {
+	Hash   string
+	Branch string
+}
+
+func (w *Webhook) Do(info webhookRepoInfo) error {
 	req, err := http.NewRequest(w.Method, w.URL, nil)
+	req.Header.Set("Git-Sync-Branch", info.Branch)
```

Comment:
nit: "Gitsync" since that is the proper name of this tool

Reply:
Ok.